--- origsrc/SimpleGNUChess-7/chess.py	2012-10-27 08:15:26.000000000 -0500
+++ src/SimpleGNUChess-7/chess.py	2013-01-20 20:16:22.854586900 -0600
@@ -64,7 +64,6 @@ class Gnuchess():
                  colors=['#A0FFA0', '#FF8080']):
         self._activity = parent
         self._bundle_path = path
-        self._bin_path = 'bin/i686'
         self._colors = ['#FFFFFF']
         self._colors.append(colors[0])
         self._colors.append(colors[1])
@@ -116,20 +115,12 @@ class Gnuchess():
         # Generate the sprites we'll need...
         self._sprites = Sprites(self._canvas)
         self._generate_sprites(colors)
-
-        p = subprocess.Popen(['uname', '-p'],
-                             stdin=subprocess.PIPE,
-                             stdout=subprocess.PIPE,
-                             stderr=subprocess.STDOUT)
-        self._bin_path = 'bin/%s' % (BIN[p.communicate()[0].replace('\n', '')])
         self._all_clear()
 
     def move(self, my_move):
         ''' Send a command to gnuchess. '''
         # Permisos para jugar
-	os.system('chmod -R 755 bin')
-        p = subprocess.Popen(['%s/%s/gnuchess' % (self._bundle_path,
-                                                   self._bin_path)],
+        p = subprocess.Popen(['gnuchess'],
                              stdin=subprocess.PIPE,
                              stdout=subprocess.PIPE,
                              stderr=subprocess.STDOUT)
